mlflow/langchain/api_request_parallel_processor.py

```diff
@@ -191,20 +191,15 @@ def call_api(
                     {"page_content": doc.page_content, "metadata": doc.metadata} for doc in docs
                 ]
             elif isinstance(self.lc_model, lc_runnables_types()):
-                (
-                    prepared_request_json,
-                    did_perform_chat_conversion,
-                ) = APIRequest._transform_request_json_for_chat_if_necessary(
-                    self.request_json, self.lc_model
-                )
                 if isinstance(self.request_json, dict):
                     # This is a temporary fix for the case when spark_udf converts
                     # input into pandas dataframe with column name, while the model
                     # does not accept dictionaries as input, it leads to errors like
                     # Expected Scalar value for String field 'query_text'
                     try:
                         response = self.lc_model.invoke(
```

Comment:
Should we raise Exception here if prepared_request_json is a list? As the invoke will take the first item 🤔 but the result is not correct